.gitmodules

```diff
@@ -0,0 +1,6 @@
+[submodule "apps/wfo-ui-surf"]
+	path = apps/wfo-ui-surf
+	url = git@git.ia.surfsara.nl:netdev/automation/projects/orchestrator-ui.git
```

Comment:
I do know this is the only way for now, BUT is this link to a surf-repo something we want to keep in the public facing repo?

Reply:
No, fixed and to be discussed